src/Test/Verilog/SVType.idr

```diff
@@ -57,18 +57,18 @@ namespace States
     (==) S4 S4 = True
     (==) _  _  = False
 
-namespace IntegerAtomType
+namespace Vector
 
   ||| 6.9 Vector declarations
   ||| A data object declared as reg, logic, or bit (or as a matching user-defined type or implicitly as logic)
   ||| without a range specification shall be considered 1-bit wide and is known as a scalar. A multibit data object
   ||| of one of these types shall be declared by specifying a range and is known as a vector. Vectors are packed
   ||| arrays of scalars
   public export
-  data IntegerAtomType = Byte' | Shortint' | Int' | Longint' | Integer' | Time';
+  data Vector = Byte' | Shortint' | Int' | Longint' | Integer' | Time';
 
   public export
-  Eq IntegerAtomType where
+  Eq Vector where
```

Comment:
Since Vector, Atom, and Real are fundamental concepts in SystemVerilog Type System, should we add some short comments describing briefly the differences? It is less needed by Real as for now it is self-explanatory, but maybe we should provide a reasoning why Atoms and Vectors are so essentially different (i.e. directly mention packed array and corresponding `PABasic` type)

Reply:
Okay, that's a good point too